modules/oe_webtools_media/oe_webtools_media.install

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Install, uninstall and update hooks for OpenEuropa Webtools Media module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\File\Exception\FileException;
+use Drupal\Core\File\FileSystemInterface;
+
+/**
+ * Implements hook_install().
+ *
+ * @see \media_install()
+ */
+function oe_webtools_media_install() {
+  $source = drupal_get_path('module', 'oe_webtools_media') . '/images/icons';
+  $destination = \Drupal::config('media.settings')->get('icon_base_uri');
+  /** @var \Drupal\Core\File\FileSystemInterface $file_system */
+  $file_system = \Drupal::service('file_system');
+  $file_system->prepareDirectory($destination, FileSystemInterface::CREATE_DIRECTORY | FileSystemInterface::MODIFY_PERMISSIONS);
+
+  $files = $file_system->scanDirectory($source, '/.*\.(svg|png|jpg|jpeg|gif)$/');
```

Comment:
Reading the ticket's comments the svg files cannot be styled but we still search for svg extensions. Can we keep only the extensions that we can use?